### ISSUE TITLE
A11y_Visual Studio Code Services_Home_Home_Reflow: On applying reflow resolution(320*256) two scroll bars are appearing for whole page.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#6207]
### GitHub Tags
#WCAG1.4.10; #A11ySev3; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #Reflow;

### Environment Details
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 111.0.5563.147 (Official Build) (64-bit)

### Pre Requisites

1. Open the web page / screen of the web application that is to be tested
2. Open Chrome/Edge dev Inspect tools. Keyboard shortcut "Ctrl+Shift+I"
3. Ensure devtools are customized to Dock location: "undock into separate window". Minimize devtools so they are not obscuring the page.
4. Zoom the browser window of the page to be tested to 200%.
5. Adjust the browser window's height and width to the required viewport of 320px*256px. As long as devtools remain open while resizing the viewport size will be displayed in the upper right corner of the browser so the size can be verified.

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Verify whether on applying reflow resolution(320*256) two scroll bars are appearing for whole page or not.

### Actual Result
On applying reflow resolution(320*256) two scroll bars are appearing for whole page.

### Expected Result
On applying reflow resolution, only one scroll bar either horizontal or vertical should be provided for the page.

### User Impact
People with low display resolution may find difficulty in viewing the page and will miss the important content of the page.

### Attachment
![Bug_Reflow](https://user-images.githubusercontent.com/99243079/230548395-bcc13a9d-a78c-4015-bd3c-1a95c89863bd.gif)

**Comment by rzhao271** [comment]
Hi, this issue should be resolved now.

**Comment by Samhitha66** [comment]
Regressed the issue in the below environment
URL:https://code.visualstudio.com/
The issue is fixed. Hence closing the bug.
Please find the attachment for the reference.
#Regressed:05-10-23; #Closed;
![image](https://github.com/microsoft/vscode-docs/assets/99243079/1d5c721f-74ba-4af5-9597-3cbb24095110)
